crates/gitql-ast/src/function.rs

```diff
@@ -371,6 +380,15 @@ fn text_translate(inputs: Vec<Value>) -> Value {
 
     return Value::Text(text);
 }
+fn text_unicode(inputs: Vec<Value>) -> Value {
+    let n = if let Some(c) = inputs[0].as_text().chars().next() {
+        c as u32
+    } else {
+        return Value::Integer(0);
+    };
+
+    return Value::Integer(n.into());
+}
```

Comment:
Your code looks good but we can make it more similar to read like below, think of it as return result or default value 

```rs
if let Some(c) = inputs[0].as_text().chars().next() {
   return Value::Integer((c as u32).into());
}
return Value::Integer(0);
```

Reply:
thanks for the feedback. Fixed!

Reply:
Good thanks